cms/templates/cms/home_page.html

```diff
@@ -9,54 +9,156 @@
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
 
 {% block content %}
-<div id="public">
-    <div class="purpose-statement-box">
-      <div class="purpose-statement-text"
-    {% if page.title_background %}
-      {% image page.title_background original as title_background %}
-           style="background-image: url({{ title_background.url }})"
+
+<div id="header">
+  <div class="header-logo">
+    <a href="/">
+      <img src="/static/images/mit-micromasters.png" alt="MIT micromasters">
+    </a>
+  </div>
+  <div class="header-links-dashboard">
+    <a href="/dashboard">Dashboard</a>
+  </div>
+  <div class="header-links-login">
+    {% if authenticated %}
+    <a class="btn btn-outline btn-success" href="/logout">
+      Sign out
+    </a>
+    {% else %}
+    <a class="btn btn-outline btn-success" href="{% url 'social:begin' 'edxorg' %}">
+      Sign in
+    </a>
     {% endif %}
-      >
-        <h1>{{ page.title }}</h1>
+  </div>
+</div>
 
-        <button type="button" class="learn-more-btn button" >Learn more</button>
-      </div>
+<div class="main-content-wrapper app-media">
+  <section class="main-content">
 
-    </div>
-    <div class="login-button-wrapper">
-      {% if authenticated %}
-      <h1>Welcome, {{ username }}</h1>
-      {% else %}
-      <a class="login-button button" href="{% url 'social:begin' 'edxorg' %}">Log in with edx.org</a>
-      {% endif %}
-    </div>
-    <div class="programs-container">
-        {% for program in programs %}
-          <div class="program-box">
-            <span style="line-height: 115px"></span>
-            {% if program.programpage %}
-            <a href="{{ program.programpage.url }}">
-              <div class="program-name">{{ program }}</div>
+
+    <div class="banner-wrapper">
+      <div class="banner-wrapper-content">
+        <h1>
+          MIT credentials are now attainable by anyone in the world with the discipline, dedication, and intelligence to master course material.
+        </h1>
+        <a href="add" role="button" class="btn btn-primary btn-lg btn-learn-more">
+          <span>LEARN MORE</span>
+        </a>
+      </div>
+      <div class="banner-bottom">
+        <div class="banner-wrapper-content">
+          <h2>Get Started Today!
+            <a href="add" role="button" class="btn btn-success sign-in-button margin-left-10">
```

Comment:
the `href` for this button is wrong: but I am not sure what we should do in case the user is already signed in. 
For the moment let's just show it also when the user is logged in.